dev/io.openliberty.persistence.3.1.thirdparty/resources/org/eclipse/persistence/exceptions/i18n/ValidationExceptionResource.nlsprops

```diff
@@ -633,9 +633,12 @@
 
 7356=Procedure: [{1}] cannot be executed because {0} does not currently support multiple out parameters
 
-7357=The [{0}] URL for the [{1}] resource does not belong to a valid persistence root, as defined by the JPA Specification.
+7357=The "[{0}]" URL for the "[{1}]" resource does not belong to a valid persistence root, as defined by the Jakarta Persistence specification
 
 7358=Incorrect ASM service name provided.
 
 7359=No any ASM service available.
 
+7360=Database password was encrypted by deprecated algorithm.
```

Comment:
The way this is coded, it will be 2 separate message, but in EclipseLink, it will be a single message with a newline character.  To add a newline in a .nlspops file, the correct way is "\n\", but that will be flagged as not matching EclipseLink.

For now, I recommend just removing the "newline", and making this message all on one line. Hopefully EclipseLink will accept the change to remove the newline, along with the other suggestions.

Reply:
Oh, it would also be good to check all of these message and make sure the wording matches what was delivered for JPA 3.2 recently.  This one was also part of JPA 3.2, so we should be consistent in the way we have modified what came over from EclipseLink.